samples/boot/oauth2resourceserver-multitenancy/src/main/resources/application.yml

```diff
@@ -3,6 +3,14 @@ spring:
     oauth2:
       resourceserver:
         jwt:
+          issuer-uri: ${mockwebserver.url}
           jwk-set-uri: ${mockwebserver.url}/.well-known/jwks.json
+        multi-tenant-jwt:
```

Comment:
I did not have mock server that supports JWTs with iss claims hence have put these as sample